src/com/facebook/buck/testrunner/TestNGRunner.java

```diff
@@ -134,6 +141,79 @@ public void transform(ITestAnnotation annotation, Class testClass, Constructor t
     }
   }
 
+  /**
+   * Listens for suite configuration / failure, captures stdout/stderr when configuring for
```

Comment:
I see. I saw in other places when `/` is used for or, there is not spaces between the words and the `/`. Either removing the spaces or changing to `or` would be fine I feel.